File: CTDK_Python/118/_118.py
n = int(input("Nhap n: "))
if (n == 1):
    ahh = 2
else:
    at = 2
    i = 2

    while (i <= n):
        ahh = (-9 * at - 24) / (5 * at + 13)
        i += 1
        at = ahh

print("Ket qua:", ahh)

File: CTDK/118/118.cpp
#include <iostream>
using namespace std;

int main()
{
	int n;
	cout << "Nhap n: ";
	cin >> n;

	float ahh = 0;
	float at = 2;
	int i = 2;
	while (i <= n)
	{
		ahh = (-9 * at - 24) / (5 * at + 13);
		i++;
		at = ahh;
	}
	cout << "So hang thu " << n << " : " << ahh << endl;
	return 0;
}